client/src/main/scala/drt/client/components/Navbar.scala

```diff
@@ -1,48 +1,74 @@
 import drt.client.modules.GoogleEventTracker
 import drt.client.services.SPACircuit
-import io.kinoplan.scalajs.react.material.ui.core.MuiSnackbar
+import drt.client.services.handlers.{CloseFeatureGuideDialog, FeatureGuideDialog, GetFeatureGuides, GetViewedFeatureIds, IsNewFeatureAvailable}
 import io.kinoplan.scalajs.react.material.ui.core.internal.Origin
+import io.kinoplan.scalajs.react.material.ui.core.{MuiBadge, MuiSnackbar}
 import japgolly.scalajs.react.component.Scala.{Component, Unmounted}
 import japgolly.scalajs.react.extra.router.{BaseUrl, RouterCtl}
 import japgolly.scalajs.react.vdom.html_<^._
-import japgolly.scalajs.react.{Callback, CtorType, ReactEvent, ScalaComponent}
+import japgolly.scalajs.react.{BackendScope, Callback, CtorType, ReactEvent, ScalaComponent}
 import org.scalajs.dom
 import uk.gov.homeoffice.drt.auth.LoggedInUser
 import uk.gov.homeoffice.drt.feeds.FeedSourceStatuses
 import uk.gov.homeoffice.drt.ports.AirportConfig
+import uk.gov.homeoffice.drt.training.FeatureGuide
 
-case class NavbarModel(feedStatuses: Pot[Seq[FeedSourceStatuses]], snackbarMessage: Pot[String])
-
+case class NavbarModel(feedStatuses: Pot[Seq[FeedSourceStatuses]],
+                       snackbarMessage: Pot[String],
+                       trainingDataTemplates: Pot[Seq[FeatureGuide]],
+                       toggleDialog: Pot[Boolean],
+                       featureGuideViewIds: Pot[Seq[String]])
 
 object Navbar {
-  case class Props(ctl: RouterCtl[Loc],
-                   page: Loc,
-                   loggedInUser: LoggedInUser,
-                   airportConfig: AirportConfig)
+  case class Props(
+                    ctl: RouterCtl[Loc],
+                    page: Loc,
+                    loggedInUser: LoggedInUser,
+                    airportConfig: AirportConfig)
 
   case class State(showDropDown: Boolean)
 
   def handleClose: (ReactEvent, String) => Callback = (_, _) => {
     Callback(SPACircuit.dispatch(SetSnackbarMessage(Empty)))
   }
 
-  val component: Component[Props, State, Unit, CtorType.Props] = ScalaComponent.builder[Props]("NavBar")
-    .initialState(if (dom.window.innerWidth > 768) State(true) else State(false))
-    .renderPS { (scope, props, state) =>
-      val rcp = SPACircuit.connect(m => NavbarModel(m.feedStatuses, m.snackbarMessage))
 
+  class Backend($: BackendScope[Props, State]) {
+    val rcp: ReactConnectProxy[NavbarModel] = SPACircuit
+      .connect(m => NavbarModel(m.feedStatuses, m.snackbarMessage, m.featureGuides, m.toggleDialog, m.featureGuideViewedIds))
+
+    def handleOpenDialog(e: ReactEvent): Callback = {
+      e.preventDefaultCB >>
+        Callback(SPACircuit.dispatch(FeatureGuideDialog(true)))
+    }
+
+    def handleDialogClose(e: ReactEvent): Callback = {
+      val closeDialog = Callback(SPACircuit.dispatch(CloseFeatureGuideDialog()))
+      e.preventDefaultCB >> closeDialog
+    }
+
+    def handleBadgeCount(viewedFeatureIds: Seq[String], templateFeatureIds: Seq[String]): Int = {
```

Comment:
This is returning the badge count so I would remove 'handle' and just call it `badgeCount`